exasol_script_languages_container_ci/lib/github_release_asset_uploader.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from github import Github
+
+
+class GithubReleaseAssetUploader(object):
+    """
+    Implements upload to a Github Release.
+    See https://docs.github.com/en/rest/releases/assets#upload-a-release-asset for details.
+    The access token needs to be stored in the environment variable GITHUB_TOKEN
+    """
+    def __init__(self):
+        self._token = os.getenv("GITHUB_TOKEN")
```

Comment:
Should we really inject the token via environment variable here? I maybe would do it at the caller site.

Reply:
Or alternatively, inject the whole PyGithub object, this should allow you mock injection

Reply:
The `uploader` is already injected, and mocked in the tests. I added the github token now as parameter.